dolphinscheduler-dao/src/main/resources/sql/upgrade/3.2.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -53,8 +53,8 @@ BEGIN
 ALTER TABLE t_ds_process_definition ADD COLUMN `execution_type` tinyint(4) DEFAULT '0' COMMENT 'execution_type 0:parallel,1:serial wait,2:serial discard,3:serial priority';
 END IF;
 END;
- d//
- delimiter ;
+d//
+delimiter ;
```

Comment:
are those changes for bug fixes or just formatted?

Reply:
This is a bug for current parser, I fix this in parser and rollback this file.